### PR TITLE
Evaluate tab clients client-scopes

### PR DESCRIPTION
**Pull request by edewit**
## Motivation

creates a tab as descriped here: [marvel app](https://marvelapp.com/prototype/5cc3de7/screen/71967641)

## Brief Description

Adds the evaluate tab under client-scope of the clients.

## Verification Steps

Select a client then go to the client-scope tab and sub tab "evaluate"

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated
- [x] Formatting has been performed via prettier/eslint
- [x] Type checking has been performed via 'yarn check-types'

### CODE REVIEW

**Comment by edewit** [comment]
@srambach could you have a look at the tabs, currently it looks like this:
image

**Comment by ssilvert** [comment]
@edewit Since it says WIP, is this a PR that you intend to merge or are you just trying to get feedback from @srambach ?

**Comment by edewit** [comment]
Yes I wanted @srambach to take a look at how to put the tabs on the right of the screen. I could do it myself, but I'm sure she will come up with something a lot better

**Comment by ssilvert** [comment]
> Yes I wanted @srambach to take a look at how to put the tabs on the right of the screen. I could do it myself, but I'm sure she will come up with something a lot better

OK.  I won't review this one then.

**Comment by srambach** [comment]
@edewit OK, what I think you will need to do is to use the "separate content" sort of tabs. https://www.patternfly.org/v4/components/tabs#separate-content
and
add a grid to put the tabs side by side with the tab contents.

Something like:
`
<Grid>
  <GridItem span={8}>
    <Form />
    <TabContent id="content1" />
    <TabContent id="content2" />
    <TabContent id="content3" />
  </GridItem>
  <GridItem span={4}>
    <Tabs>
      <Tab title="title1" eventKey={0} tabContentId="content1"></Tab>
      <Tab title="title2" eventKey={1} tabContentId="content2"></Tab>
      <Tab title="title3" eventKey={2} tabContentId="content3"></Tab>
    </Tabs>{" "}
  </GridItem>
</Grid>
`

**Comment by edewit** [comment]
@srambach when I use the `<TabContent` with the refs the page stops working

File: src/clients/scopes/EvaluateScopes.tsx [hunk 1]
```diff
@@ -1,64 +1,393 @@
...
+              setIsUserOpen(false);
+            }}
+            isOpen={isUserOpen}
+          >
+            {userItems}
+          </Select>
+        </FormGroup>
+      </Form>
+      <Grid className="keycloak__scopes_evaluate__tabs">
+        <GridItem span={8}>
```
Comment (srambach) [inline src/clients/scopes/EvaluateScopes.tsx:325]: 2 things to fix the horizontal spacing between the vertical tabs and the table - add `hasGutter` to the Grid and add padding to the GridItem containing the tabs `className="pf-u-pl-xl"`

**Comment by srambach** [comment]
Looks like there is help text missing according to the mockup. I think the vertical spacing is off between the tabs  (Setup | Evaluate) and the content but the help text will affect this also.

Axe shows invalid aria-labelledby values in a couple of spots.

**Comment by edewit** [comment]
@srambach fixed the space between the tabs and the tab content, but I can't seem to be able to fix the aria labels

File: src/clients/scopes/EvaluateScopes.tsx (hunk 1 above)
Comment (srambach) [reply src/clients/scopes/EvaluateScopes.tsx]: The mockup shows 40px here, so you need both the gutter and the padding. I'm seeing just the 16px gutter right now.

**Comment by srambach** [comment]
> @srambach fixed the space between the tabs and the tab content, but I can't seem to be able to fix the aria labels

The two tooltips and the copy button should be straightforward. 
On the vertical tabs, is it a problem with the component? If so, please put an issue on Patternfly so that it can be addressed.

**Comment by ssilvert** [comment]
@edewit This is using useFetch, so it no longer works at all.  If you can go ahead and fix that plus remaining issues from @srambach then we can merge Tuesday.

**Comment by edewit** [comment]
@srambach fixed the 2 two tooltips, but still don't know how to fix the copy paste or the tabs.
@ssilvert rebased

**Review by ssilvert — APPROVED** [review]

**Pull request merged**